### PR TITLE
Update the changelog and build properties for version 2.8.0

### PR DESCRIPTION
**Pull request by tvbeek**
Type: documentation update  
Issue: -
Breaking no

This PR prepares the release of 2.8.0

If approved I did plan to release a new version on monday.

### CODE REVIEW

**Comment by tvbeek** [comment]
I updated fix with fixed. If we add more changes it can be added to the changelog.

**Comment by sveneld** [comment]
When version 2.8.0 will be released?

**Comment by tvbeek** [comment]
@sveneld that is a good question. We want to add https://github.com/pdepend/pdepend/pull/479 and https://github.com/pdepend/pdepend/pull/478 to this release. So we are waiting on that PR's to be finished.

**Comment by ravage84** [comment]
@tvbeek I don't think the list is complete.
I compared the [tag of our last release, 2.7.1 with the current master](https://github.com/pdepend/pdepend/compare/2.7.1...master).
It contains a lot more issues and PRs. Is that intended?

**Comment by ravage84** [comment]
These are [the commits of the last release, 2.5.0 made by Maneul Pichler](https://github.com/pdepend/pdepend/commits/2.5.0).

- [Release 2.5 prepared](https://github.com/pdepend/pdepend/commit/a1f1b0d2abd6a1b57f8c98911a98599849dd5567)
  - Changelog
  - Build properties
  - changes.xml 
- [Blog post about release 2.5.0 added](https://github.com/pdepend/pdepend/commit/0c50874333149c0dad5a2877801aed148f2767ff)
  - .blog.xml
  - news/pdepend-2.5.0-released.rst

Have we changed/simplified the release process so much?

**Comment by tvbeek** [comment]
> 
> 
> @tvbeek I don't think the list is complete.
> I compared the [tag of our last release, 2.7.1 with the current master](https://github.com/pdepend/pdepend/compare/2.7.1...master).
> It contains a lot more issues and PRs. Is that intended?

I don't intend to exclude information from the changelog unless I think it isn't useful for the users.😃  
I suspect that I intended to exclude the Tidelift Pr's but they can be called in one line.
And #484,  #487, #478 and #485 where added after the last change of this PR.

I updated the changelog with the new changes.

**Comment by tvbeek** [comment]
> 
> 
> These are [the commits of the last release, 2.5.0 made by Maneul Pichler](https://github.com/pdepend/pdepend/commits/2.5.0).
> 
>     * [Release 2.5 prepared](https://github.com/pdepend/pdepend/commit/a1f1b0d2abd6a1b57f8c98911a98599849dd5567)
>       
>       * Changelog
>       * Build properties
>       * changes.xml
> 
>     * [Blog post about release 2.5.0 added](https://github.com/pdepend/pdepend/commit/0c50874333149c0dad5a2877801aed148f2767ff)
>       
>       * .blog.xml
>       * news/pdepend-2.5.0-released.rst
> 
> 
> Have we changed/simplified the release process so much?

Good point, 

I think we forgot changes.xml and the news on the website for the last releases, that need to be added (and also for the other releases after 2.5.0)

**Comment by ravage84** [comment]
> I think we forgot changes.xml and the news on the website for the last releases, that need to be added (and also for the other releases after 2.5.0)

@tvbeek yeah, probably. Are you going to address that in this PR or in another one?

By the way the whole process is undocumented.
Could you add a Wiki page for that, please?
I would include the points from my list but also the work done around, which you/we do normally, like finding a release date etc.

**Comment by tvbeek** [comment]
> 
> 
> > I think we forgot changes.xml and the news on the website for the last releases, that need to be added (and also for the other releases after 2.5.0)
> 
> @tvbeek yeah, probably. Are you going to address that in this PR or in another one?

I was currently thinking about the best option. For this version it could be in this PR. But for the other versions it should be more clear to do it in another PR.

> 
> By the way the whole process is undocumented.
> Could you add a Wiki page for that, please?
> I would include the points from my list but also the work done around, which you/we do normally, like finding a release date etc.
Yes, I will document it

**Comment by ravage84** [comment]
> I was currently thinking about the best option. For this version it could be in this PR. But for the other versions it should be more clear to do it in another PR

I'm fine either way. Just tell us, when you are ready for a final review.

> Yes, I will document it

Thanks! :bow:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+- Added [\#466](https://github.com/pdepend/pdepend/pull/466) Allow to configure file cache ttl in configuration file. (Issue [\#465](https://github.com/pdepend/pdepend/issues/465))
+- Added [\#487](https://github.com/pdepend/pdepend/pull/484) Support arrow functions return type hints.
+- Fixed [\#467](https://github.com/pdepend/pdepend/pull/467) Use the HOME env variable also for Windows. (Issue [\#447](https://github.com/pdepend/pdepend/issue/447) )
+- Fixed [\#475](https://github.com/pdepend/pdepend/pull/475) Fixed [\#474](https://github.com/pdepend/pdepend/issues/474) typed property visibility 
+- Fixed [\#476](https://github.com/pdepend/pdepend/pull/476) Fixed [\#473](https://github.com/pdepend/pdepend/issues/473) handle skipped variable in destructuring array
+- Fixed [\#478](https://github.com/pdepend/pdepend/pull/478) Fixed [\#472](https://github.com/pdepend/pdepend/issues/472) Use the current token instead of the first inner one to determine the class start.
+- Fixed [\#483](https://github.com/pdepend/pdepend/pull/483) Fixed bool flip in parseEscapedAstLiteralString()
```
Comment (kylekatarnls) [inline CHANGELOG.md:11]:
```suggestion
- Fixed [\#483](https://github.com/pdepend/pdepend/pull/483) Fixed bool flip in parseEscapedAstLiteralString()
- Fixed [\#475](https://github.com/pdepend/pdepend/pull/479) Fixed [\#299](https://github.com/pdepend/pdepend/issues/299) Class constant expression declaration.
```

**Pull request merged**